src/components/misc/InfoList.jsx

```diff
@@ -16,10 +17,17 @@ export default class InfoList extends React.Component {
                     let className = 'InfoListItem InfoListItem-' + item.name;
                     let value = item.value;
 
-                    if (item.msgId) {
+                    // use custom value if set, using the msgId as a fallback
+                    if (!value && item.msgId) {
                         value = this.props.intl.formatMessage({ id: item.msgId }, item.msgValues);
                     }
 
+                    if (value && (item.onClick || item.href)) {
```

Comment:
Great solution! Now that you've implemented this, please remove `onClick` on line 37. We discovered yesterday that `item.onClick` was ignored, and I added it to line 37 as a quick fix.

Reply:
Thanks, done!